group_vars/location_k12/networks.yml

```diff
@@ -0,0 +1,137 @@
+---
+ipv6_prefix: '2001:bf7:760:2a00::/56'
+
+# got following prefixes:
+# Router:  10.31.158.0/24
+# --DHCP:  10.31.158.0/25
+# --MESH:  10.31.158.128/26
+# --MGMT:  10.31.158.192/26
+
+networks:
+# DHCP
+  - vid: 40
+    role: dhcp
+    inbound_filtering: true
+    enforce_client_isolation: true
+    prefix: 10.31.158.0/25
+    ipv6_subprefix: 0
+    assignments:
+      k12-core: 1
+
+# MESH - 5 GHz 802.11s
+  - vid: 20
+    role: mesh
+    name: mesh_5ghz
+    prefix: 10.31.158.128/32
+    ipv6_subprefix: -1
+    mesh_ap: k12-core
+    mesh_radio: 11a_standard
+    mesh_iface: mesh
+
+# MESH - 2.4 GHz 802.11s
+  - vid: 21
+    role: mesh
+    name: mesh_2ghz
+    prefix: 10.31.158.129/32
+    ipv6_subprefix: -2
+    # make mesh_metric(s) for 2GHz worse than 5GHz
+    mesh_metric: 1024
+    mesh_metric_lqm: ['default 0.8']
+    mesh_ap: k12-core
+    mesh_radio: 11g_standard
+    mesh_iface: mesh
+
+# MESH - 5 GHz 802.11s - ap1
+  - vid: 22
+    role: mesh
+    name: mesh_ap1_5
+    prefix: 10.31.158.130/32
+    ipv6_subprefix: -3
+    mesh_ap: k12-ap1
+    mesh_radio: 11a_standard
+    mesh_iface: mesh
+
+# MESH - 2.4 GHz 802.11s - ap1
+  - vid: 23
+    role: mesh
+    name: mesh_ap1_2
+    prefix: 10.31.158.131/32
+    ipv6_subprefix: -4
+    # make mesh_metric(s) for 2GHz worse than 5GHz
+    mesh_metric: 1024
+    mesh_metric_lqm: ['default 0.8']
+    mesh_ap: k12-ap1
+    mesh_radio: 11g_standard
+    mesh_iface: mesh
+
+# MESH - 5 GHz 802.11s - cpe
+  - vid: 24
+    role: mesh
+    name: mesh_cpe
+    prefix: 10.31.158.132/32
+    ipv6_subprefix: -5
+    mesh_ap: k12-cpe
+    mesh_radio: 11a_standard
+    mesh_iface: mesh
+
+# MESH - segen
+  - vid: 11
+    role: mesh
+    name: mesh_segen
+    prefix: 10.31.158.133/32
+    ipv6_subprefix: -6
+
+# MESH - 5 GHz 802.11s - ap2
+  - vid: 25
+    role: mesh
+    name: mesh_ap2_5
+    prefix: 10.31.158.134/32
+    ipv6_subprefix: -7
+    mesh_ap: k12-ap2
+    mesh_radio: 11a_standard
+    mesh_iface: mesh
+
+# MESH - 2.4 GHz 802.11s - ap1
+  - vid: 26
+    role: mesh
+    name: mesh_ap2_2
+    prefix: 10.31.158.135/32
+    ipv6_subprefix: -8
+    # make mesh_metric(s) for 2GHz worse than 5GHz
+    mesh_metric: 1024
+    mesh_metric_lqm: ['default 0.8']
+    mesh_ap: k12-ap2
+    mesh_radio: 11g_standard
+    mesh_iface: mesh
+
+# MGMT
+  - vid: 42
+    role: mgmt
+    prefix: 10.31.158.192/26
+    gateway: 1
+    dns: 1
+    ipv6_subprefix: 1
+    assignments:
+      #10.31.158.193/32
```

Comment:
Is a POE switch used? If so maybe it should have a static mgmt ip. If not, please disregard this comment.

Reply:
No fancy stuff like POE switches here...